fpa/jsonauditor/jsonauditor.go

```diff
@@ -1,62 +0,0 @@
-package jsonauditor
```

Comment:
The concept of _Auditor_ was deleted... just because it wasn't something completely functional yet, so preferred to keep things clean.
The idea will exist anyway, but I suspect will have another form... I suspect it should be something that wraps the _FastAPI_ instead of being a dependency. Mostly how middlewares are attached to HTTP handlers to add the functionality of logging instead of getting in their logic which is a bit annoying.

The idea is to have some metrics on how many times cids are added, retrieved, and a bit of audit about a FastAPI usage. I'll think more about it in the next PR, happy to hear ideas too.